src/blr/Snakefile

```diff
@@ -247,15 +247,73 @@ rule symlink_reference_variants:
         "ln -s {config[reference_variants]} {output}"
 
 
-rule call_variants_freebayes:
+rule add_RG_tag:
```

Comment:
The RG tag is only needed for the GATK software right? Do you know if it affects the other variant callers? If it does not affect them i would suggest to just incorporate all these edits into the `tagbam` script. That way we don't have to read the file one more time. 

This also becomes handy if we want to merge multiple runs after mapping, such as for the SM libs I run before christmas.

Reply:
Alternatively you could also use `samtools addreplacerg`.

Something like this should work.

```
samtools addreplacerg -r ID:1 -r LB:lib1 -r PU:unit1 -r SM:20 -r PL:ILLUMINA -O BAM in.bam > out.bam
```

Reply:
Went with samtools since it involves less maintenance.